src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
```

Comment:
add fallback fonts, check out point in checklist - https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md#css